### DISCUSSION TITLE
Source Twilio - Pipeline sync failure with S3 csvnorm destination

### DISCUSSION THREAD

**Discussion by WisamulHaque** [airbytehq/airbyte#34139] [General]
I have created a connection using the following:
Source: Twilio
Destination: S3 csvnorm

Sync failure:

![image](https://github.com/airbytehq/airbyte/assets/106967803/eb8db8ac-fe15-4055-ba6f-dbc6087a7e68)
![image](https://github.com/airbytehq/airbyte/assets/106967803/5d66ecdc-b9da-4496-b542-25761cad27a6)

Logs are attached below:
[twiliocsvn_logs.txt](https://github.com/airbytehq/airbyte/files/13898667/twiliocsvn_logs.txt)

**Comment by sajarin** [comment]
hey @WisamulHaque can you confirm whether that file exists? The error is returning from the API and the API call is reporting that the file does not exist.

[a comment by an automated account is not shown]
